src/web/client/telemetry/webExtensionTelemetry.ts

```diff
@@ -86,7 +86,9 @@ export class WebExtensionTelemetry {
         duration?: number,
         errorMessage?: string,
         eventName?: string,
-        status?: string) {
+        status?: string,
+        methodName?: string) {
```

Comment:
this function signature update is correct - but we should make sure we are passing an actual methodname here - function.name property in javascript